eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Scripts.hs

```diff
@@ -33,16 +33,18 @@ module Cardano.Ledger.Alonzo.Scripts (
 
   -- * Cost Model
   CostModel,
+  CostModelError (..),
+  emptyCostModels,
   mkCostModel,
+  mkCostModels,
```

Comment:
A pity we have to expose both, especially since `mkCostModel` is meant to be the "smart constructor" (I think it's written in some doc). Now we have an even smarter one, a genius constructor :grin: 
But we need them in tests, which ultimately is a good thing!

Reply:
yea, can you think of anything better to do?

maybe I just need more documentation to make it clear when you might use each one? `mkCostModel` is when you are _serious_ :neutral_face: about needing a given cost model. and `mkCostModels` is much more laid back :relaxed:, happy to take the good things and recording the bad things for future contemplation.

Reply:
sounds good to me!